src/extension_instance.ml

```diff
@@ -125,6 +125,63 @@ let check_ocaml_lsp_available sandbox =
           current sandbox.")
 ;;
 
+let suggest_to_run_dune_pkg_lock t =
+  let open Promise.Syntax in
+  let (_ : unit Promise.t) =
+    let+ maybe_choice =
+      Window.showWarningMessage
+        ~message:
+          "Dune Package Manager is detected but no lock file is present. Do you want to \
+           run `dune pkg lock`"
```

Comment:
"Dune Package Manager is selected as the active sandbox, but no lock file is present. Do you want to  run `dune pkg lock`?"